BankManagerConsoleApp/BankManagerConsoleApp/Model/BankCustomer.swift

```diff
@@ -5,23 +5,57 @@
 //  Created by Eddy, dudu on 2022/04/26.
 //
 
+enum Banking: CaseIterable {
+    case deposit
+    case loan
+}
+
+extension Banking {
+    var name: String {
+        switch self {
+        case .deposit:
+            return "예금"
+        case .loan:
+            return "대출"
+        }
+    }
+
+    var processTime: Double {
+        switch self {
+        case .deposit:
+            return 0.7
+        case .loan:
+            return 1.1
+        }
+    }
+}
+
 protocol Customer {
     var waitingNumber: Int { get }
-    init(waitingNumber: Int)
+    var workType: Banking { get }
+    init(waitingNumber: Int, workType: Banking)
 }
 
 extension Customer {
     static func randomCustomers() -> [Self] {
         var customers = [Self]()
         let customerCount = Int.random(in: 10...30)
         for number in 1...customerCount {
-            customers.append(Self(waitingNumber: number))
+            customers.append(makeCustomer(number: number))
         }
 
         return customers
```

Comment:
```suggestion
        let customerCount = Int.random(in: 10...30)
        let customers = (1...customerCount).map(makeCustomer)
        return customers
```
고차함수를 사용하면 이렇게도 표현할 수 있을 것 같아요.

Reply:
```swift
    static func randomCustomers() -> [Self] {
        let customerCount = Int.random(in: 10...30)
        let customers = (1...customerCount).map(makeCustomer)

        return customers
    }
```

제안주신대로 바꿨습니다!
훨씬 깔끔해졌네요 :)